themes/openy_themes/openy_lily/sass/modules/_branch.scss

```diff
@@ -1082,3 +1082,66 @@ html.js {
     margin-right: 0.25em;
   }
 }
+
+.marker_tooltip {
+  color: $darker-grey;
+  font-family: "Helvetica Neue", Verdana, sans-serif;
```

Comment:
Accordingly to YMCA branding - Verdana should be sans serif font
Please use verdana() mixin for the font everywhere

Reply:
@podarok the mixing is only available in openy_lily btw

Reply:
![image](https://user-images.githubusercontent.com/5453109/44080682-7224dd42-9fb5-11e8-9d55-9ee42f9aa9fa.png)